README.md

```diff
@@ -87,11 +87,20 @@ Accessibility should be incorporated and tested throughout the design and develo
 
 ## Policy & Informal
 
-* **Analytics:** All of our sites should have code snippets for 18F and Digital Analytics Program Google Analytics as well as Google Webmaster Tools.
 * Agency or Organization Communications Office
 * Legislative Affairs
 * Office of General Counsel
 
+### Analytics
+
+All of our sites should have code snippets for
+
+* 18F Google Analytics (18F only)
```

Comment:
Our [Analytics Standards](https://github.com/18F/analytics-standards#18f-team-standards-for-analytics) say:

> Webpages should integrate both the 18F Google Analytics (GA) as well as the Digital Analytics Program (DAP).

so I think I was just going off of that. I defer to the Analytics Guild on this, so maybe I should just take out the list and link to those Standards?